docs/trestle_author.md

```diff
@@ -209,6 +217,10 @@ Validate the headers in markdown and metadata in drawio files.
 
 Turns off the validation of the structure of the document and only validates the yaml header structure and drawio files.
 
+#### Ignore files or folders from validation (`-ig`, `--ignore`)
+
+Provide regular expression after this flag to ignore folders and files which names match this regular expression during **validation only**.
+
```

Comment:
Should we deprecate exclude in this case?

Reply:
(mark as deprecated).

Reply:
Provided a mechanism to deprecate arguments for the future purposes

Reply:
Added a warning